NEWS

```diff
@@ -0,0 +1,84 @@
+etcd v3.0.4 (2016-07-27)
+
+  Bug fixes
+
+    - Ensure lease requests close request body to avoid unnecessary port allocation
+    - Fix v3 client to retry requests on rpctypes.ErrStopped
+    - Fix v3 watch progress notify time-out issue
+    - Fix etcd version capability check
+    - Fix Protocol Buffer compatibility issue, fix grpc-gateway
+    - Back-port grpc-gateway upstream patch to fix server-side errors
+    - Back-port go-systemd upstream patch to fix SdNotify issue
+
+  Security changes
+
+    - v2 auth can now use common name from TLS certificate when --client-cert-auth is enabled
+    - Go 1.6.3 with its security patches
+
+  Other changes
+
+    - Update grpc-gateway import paths
+    - v2 etcdctl ls command now supports --output=json
+    - Add /var/lib/etcd directory to etcd official Docker image
+    - Various test fixes
+    - Various documentation fixes
+
+  Backward compatibility
+
+    - Compatible with all previous v3.0.0+ releases
+    - github.com/gengo/grpc-gateway/runtime now becomes github.com/grpc-ecosystem/grpc-gateway/runtime
+
+
+etcd v3.0.3 (2016-07-15)
+
+  Bug fixes
+
+    - Fix SRV lookup for already-initialized members
+    - Fix raft for leader to correctly set RecentActive of followers
+    - Back-port gRPC upstream patch to fix TLS client issue
+
+  Other changes
+
+    - Revert Dockerfile to use CMD, instead of ENTRYPOINT, to support etcdctl run
```

Comment:
i think we only needs to include important user facing stuff like this in this doc. for bug fixes, and others, they are already covered in release notes.